### PR TITLE
Using Lottie Animations In Flutter

### LINKED ISSUE #5377: Using Lottie Animations in Flutter Applications
#### Proposed title of article
Using Lottie Animations in Flutter Applications

#### Proposed article introduction
A Lottie is a JSON-based animation file format. Lottie files are used by UX/UI designers to improve user experience in any platform. In this tutorial, we will be going through the use Lottie animations and animation controllers in Flutter applications.

#### Key takeaways 
- The reader will learn how to implement Lottie animations in a Flutter application.
- The reader will learn how to incorporate Lottie animations in different ways in a Flutter application.
- The reader will learn how to use animation controllers to manage Lottie animations in a Flutter application.

#### Article quality
This article goes in-depth to explain the implementation of Lottie animation in Flutter applications. It will also, as well, go through the use of animation controllers in Flutter in managing the animations' state. It will use. The article will use easy-to-understand English to ensure all readers understand this concept.

#### References
N/A

### PR DESCRIPTION
**Pull request by ianmasae**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #5377

### CODE REVIEW

**Review by mercymeave — COMMENTED** [review]
Hello @ianmasae 

Great article you got here :rocket: 

Proceeding to tech review.

File: content/articles/using-lottie-animations-in-flutter/index.md
Comment (mercymeave) [inline content/articles/using-lottie-animations-in-flutter/index.md:19]: Inform the user to do this after setting app the application or provide a stepwise process of setting up the application until the point of adding the Lottie dependency.

**Review by mercymeave — CHANGES_REQUESTED** [review]
Hello @ianmasae 
Great article mate :fist_oncoming:  Kindly work on that minor correction then we get done.

File: content/articles/using-lottie-animations-in-flutter/index.md
Comment (ianmasae) [reply content/articles/using-lottie-animations-in-flutter/index.md:19]: Okay

**Comment by mercymeave** [comment]
Hello @ianmasae 

Any progress?

**Comment by ianmasae** [comment]
Hello,

My article assumes that the reader has already installed Flutter to the
computer.  I think explaining the whole process of installing Flutter is a
whole new article that should be created.

I think I should add a line indicating this assumption with just the
Flutter version and the IDE that I have used and maybe include the official
Flutter website for those who want to install it.

Thank you.

**Comment by WanjaMIKE** [comment]
Please check your code. Some of it seem to be covered in other blog sites such as https://www.geeksforgeeks.org/flutter-lottie-animation/

[New Scan 6_38 AM Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/7791797/New.Scan.6_38.AM.Copyleaks.Report.pdf)

**Comment by ianmasae** [comment]
Hello,
Most of the code that appears plagiarised are Flutter boilerplate code or
simply, sections of code that are repeated in multiple places with little
to no variation. They cannot be changed.
I made a few changes to the article though.
Thank you.

**Review by mercymeave — APPROVED** [review]
Great article @ianmasae 

You can go over the changes made to improve future articles. Let us wait for the final review.

**Comment by Kayere** [comment]
Great article @ianmasae.
Thank you for the review @mercymeave.

**Pull request merged**
